### PR TITLE
Add type specifix prefixes - more compatibility with Etsy's Statsd

### PR DESCRIPTION
**Pull request by setaou**
Using Etsy's statsd, one can configure a different prefix for each key type (counter, timer, gauge, set). This feature is not available in Statsdcc, which is thus not fully compatible.

This pull request adds the missing feature, allowing the configuration of said prefixes through options in the config file.

### CODE REVIEW

File: src/configs/aggregator_config.cc [hunk 1]
```diff
@@ -15,6 +15,23 @@ AggregatorConfig::AggregatorConfig(const Json::Value& json)
     this->prefix = this->prefix + ".";
   }
 
+  this->prefixCounter = json.get("prefix_counter", "counters").asString();
```
Comment (sdomalap) [inline src/configs/aggregator_config.cc]: Thanks for taking care of the config's backward compatibility by setting defaults.

File: include/statsdcc/configs/aggregator_config.h [hunk 2]
```diff
@@ -33,6 +33,11 @@ class AggregatorConfig : public Config {
 
   std::string prefix;
 
+  std::string prefixCounter;
```
Comment (sdomalap) [inline include/statsdcc/configs/aggregator_config.h]: To keep it consistent with the rest of the code, can we please stick with snake_case for variable names, please.

An example: [current_ledger](https://github.com/wayfair/statsdcc/blob/0cffedcebf93aff189fca5e8b6605d2e979e6cb2/include/statsdcc/workers/aggregator/worker.h#L119)

Comment (setaou) [reply include/statsdcc/configs/aggregator_config.h]: Fixed.

**Comment by setaou** [comment]
Thanks for reviewing this PR. I have made the suggested changes.

**Pull request merged**